ruby/lib/keys_checker.rb

```diff
@@ -12,11 +12,13 @@ def compare(found, expected)
 
       return errors if found_keys.sort == expected_keys.sort
 
-      missing_keys = (expected_keys - found_keys)
+      missing_keys = (expected_keys - found_keys).reject do |key|
+        ENV['CI'] && found.class == Cucumber::Messages::Meta && key == :ci
```

Comment:
Do we want to just do this regardless of the `CI` env var? I recall your machine happened to have something set for this, but what if it doesn't?

Reply:
In hindsight yes, that's not a negative. But what it means is running **on** ci won't check this key.

But if this key is just jargon from the CI executor then it seems reasonable to just never match on it. However then we enter the realm of not testing a part of the schema ever? Thoughts?

Reply:
I would say never match on it, JS and JVM do basically the same and the `ci-environment` library does thorough enough testing of its own. Really the focal point of CCK is the messages about the tests themselves.